apps/landing-page/pages/blog/exploring-quay-registry-in-backstage.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Exploring Quay registry in Backstage
+date: 2023/02/20
+description: The Janus IDP family of Backstage plugins is expanding! Please welcome our new member - a frontend plugin that enriches application view in Backstage with insights from Quay registry.
+---
+
+import { Authors } from 'ui/components';
+
+# Exploring Quay registry in Backstage
+
+<Authors authors={['tumido_']} />
+
+The Janus IDP family of Backstage plugins is expanding! Please welcome our new member - a frontend plugin that enriches application view in Backstage with insights from the Quay registry.
+
+Backstage charts and models the software ecosystem as Catalog entities. Users compose small individual service components into a bigger picture. And in order to truly understand and fully describe the individual building blocks, Backstage users construct views to capture different aspects of these components. From technical documentation, through dependencies and relation to deployment state, CI and CD. And part of this picture is understanding what is actually being deployed into live environments: a view of the available and deployed container images. This new Quay plugin for Backstage brings that view.
+
+Quay is an OCI-compatible registry that allows users to build, store and distribute container images and other OCI artifacts. It is available as a hosted service on quay.io as well as a self-hosted environment deployable to any OpenShift cluster through a Quay operator. 
+
+
+## Installation and setup
+
+With this plugin, viewing available container images in a particular repository is easy. The following guide will elaborate in detail on individual steps and all currently supported settings and options.
+
+First, it is necessary to install the plugin. Please add it to the frontend of your Backstage instance:
+
+```bash
+yarn workspace app add @janus-idp/backstage-plugin-quay
+```
+
+### Connecting to Quay registry via Proxy
+
+Plugin leverages the Backstage native proxy capabilities to query Quay API; therefore, it needs to be configured in `app-config.yaml`. In order to connect the Backstage instance to the public hosted Quay.io environment, the following configuration can be used:
+
+```yaml filename="app-config.yaml"
+proxy:
+  '/quay/api':
+    target: 'https://quay.io'
+    changeOrigin: true
+    headers:
+      X-Requested-With: 'XMLHttpRequest'
+```
+
+When accessing private Quay repositories, it is necessary to extend this configuration with `Authorization` header and Quay API token. This token can be obtained by creating a Quay Application using the [steps outlined in this documentation](https://docs.quay.io/api/#applications-and-tokens). Once a token is obtained, the header can be set by extending the `app-config.yaml` setup above with an additional header:
```

Comment:
```suggestion
When accessing private Quay repositories, it may be necessary to extend this configuration with an `Authorization` header and Quay API token. This token can be obtained by creating a Quay Application using the [steps outlined in this documentation](https://docs.quay.io/api/#applications-and-tokens). Once a token is obtained, the header can be set by extending the `app-config.yaml` setup above with an additional header:
```